Framework/DataObjects/inc/MantidDataObjects/EventList.h

```diff
@@ -259,6 +260,8 @@ class MANTID_DATAOBJECTS_DLL EventList : public Mantid::API::IEventList {
 
   void filterByPulseTime(Types::Core::DateAndTime start, Types::Core::DateAndTime stop, EventList &output) const;
 
+  void filterByTimeROI(Kernel::TimeROI *timeRoi, EventList &output) const;
```

Comment:
Please rename the function to `filterByPulseTime`. The data structure of the first argument is less informative that the type of filtering the function performs.

Reply:
The name has been changed